apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/ConfigUtils.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+/**
+ * This helper class abstracts the source of configuration values and handles insertion of default
+ * values when none are provided by the environment. Currently implemented to pull values from
+ * environment variables.
+ */
+class ConfigUtils {
+  /**
+   * Retrieve configuration value with the given name. Returns the specified default value when
+   * there is no configuration value matching the key.
+   *
+   * @param key key that identifies the particular configuration value.
+   * @param defaultValue default value if there is no configuration value matching the key
+   * @return either the configuration value or the default
+   */
+  public static String getString(String key, String defaultValue) {
+    String value = System.getenv(key);
+    return value != null ? value : defaultValue;
+  }
+
+  /**
+   * Retrieve integer configuration value with the given name. Returns the specified default value
+   * when there is no configuration value matching the key.
+   *
+   * @param key key that identifies the particular configuration value.
+   * @param defaultValue default value if there is no configuration value matching the key
+   * @return either the configuration value or the default
+   */
+  public static int getInt(String key, int defaultValue) {
+    String strValue = System.getenv(key);
+    return strValue != null ? Integer.parseInt(strValue) : defaultValue;
```

Comment:
I prefer to fail fast in configuration rather than falling back on a default when a config file or environment variable is incorrect.  We could catch it and log some error message before rethrowing though.

Reply:
💯 on failing fast; I was thinking the latter and having a useful message in the logs that our env variable was misconfigured.

Reply:
I have pushed a commit with this change.